### PR TITLE
Sle 12 sp3 bnc 1099505

### PR DESCRIPTION
**Pull request by schubi2**

### CODE REVIEW

File: startup/YaST2.call [hunk 1]
```diff
@@ -673,6 +677,11 @@ if [ "$server_running" = 1 ];then
 	done
 fi
 
+if [ "$Y2_MODE" = "ncurses" ]; then
+    # Enalbe display of status messages on the console (systemd)
```
Comment (lslezak) [inline startup/YaST2.call]: `Enable`

File: package/yast2-installation.changes [hunk 2]
```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Wed Aug  1 16:43:45 CEST 2018 - schubi@suse.de
+
+- Disalbe display of status messages on the consol (bsc#1099505).
```
Comment (lslezak) [inline package/yast2-installation.changes]: Typos: `Disable`, `console`

File: startup/YaST2.call [hunk 3]
```diff
@@ -129,6 +129,10 @@ function prepare_for_ncurses () {
 	check_run_fbiterm
 	log "\tCheck for FB-I-terminal: RUN_FBITERM = $RUN_FBITERM"
 	set_inst_ncurses_env
+	#=============================================================
+	# Disable display of status messages on the console (systemd)
+	#-------------------------------------------------------------
+	/usr/bin/kill -SIGRTMIN+21 1
```
Comment (lslezak) [inline startup/YaST2.call]: Um, where the magic number `+21` comes from? Any docu or link?

File: startup/YaST2.call [hunk 4]
```diff
@@ -673,6 +677,11 @@ if [ "$server_running" = 1 ];then
 	done
 fi
 
+if [ "$Y2_MODE" = "ncurses" ]; then
+    # Enalbe display of status messages on the console (systemd)
+    /usr/bin/kill -SIGRTMIN+20 1
```
Comment (lslezak) [inline startup/YaST2.call]: The same here, any docu for `+20`?

File: startup/YaST2.call [hunk 5]
```diff
@@ -673,6 +679,13 @@ if [ "$server_running" = 1 ];then
 	done
 fi
 
+if [ "$Y2_MODE" = "ncurses" ]; then
```
[the remark this answers is not in the document]
Comment (schubi2) [reply startup/YaST2.call]: I will test it .....

**Pull request merged**